go/pkg/filemetadata/cache_posix_test.go

```diff
@@ -28,7 +29,7 @@ func TestExecutableCacheLoad(t *testing.T) {
 		Digest:       wantDg,
 		IsExecutable: true,
 	}
-	if diff := cmp.Diff(want, got); diff != "" {
+	if diff := cmp.Diff(want, got, cmpopts.IgnoreFields(Metadata{}, "MTime")); diff != "" {
```

Comment:
Declare "MTime" as a const and use everywhere?

Reply:
Done.